lib/resources/orders.js

```diff
@@ -49,15 +49,10 @@ module.exports = function (api) {
     },
 
     create(params = {}, callback) {
-      let isNotForm = false
       let { amount, currency, receipt, partial_payment,payment_capture, notes, method,
             ...otherParams } = params
       currency = currency || 'INR'
 
-      if(params.hasOwnProperty("first_payment_min_amount")){
```

Comment:
question: is this param `first_payment_min_amount` removed from api request?

Reply:
@dhwanilvyas earlier if the param has `first_payment_min_amount `  property then we had to send the data in `body` otherwise it's not working. [for your reference](https://github.com/razorpay/razorpay-node/blob/master/lib/api.js#L72).

but know it is working without this check, even the param has `first_payment_min_amount`